plsql/PlSqlParser.g4

```diff
@@ -911,30 +911,68 @@ create_mv_refresh
 
 create_table
     : CREATE (GLOBAL TEMPORARY)? TABLE tableview_name 
-        ( '(' (','? datatype_null_enable)+
-        (',' (CONSTRAINT constraint_name)?
-          ( primary_key_clause
-          | foreign_key_clause
-          | unique_key_clause
-          | check_constraint
-          ) constraint_state?
-        )*
-        ')' )?
-        (ON COMMIT (DELETE | PRESERVE) ROWS)?
-        physical_properties?
-        table_compression?
-// Column_properties clause goes here
-// Partition clause goes here
-         table_range_partition_by_clause?
-// Many more varations to capture
-       row_movement_clause?
-       flashback_archive_clause?
-
-      (AS subquery)?
-
+        (relational_table | object_table | xmltype_table) (AS subquery)?
       ';'
     ;
 
+xmltype_table
+    : OF XMLTYPE ('(' object_properties ')')? (XMLTYPE xmltype_storage)? xmlschema_spec? xmltype_virtual_columns? (ON COMMIT (DELETE | PRESERVE) ROWS)? oid_clause? oid_index_clause? physical_properties? column_properties? table_partitioning_clauses? (CACHE | NOCACHE)? (RESULT_CACHE '(' MODE (DEFAULT | FORCE) ')')? parallel_clause? (ROWDEPENDENCIES | NOROWDEPENDENCIES)? (enable_disable_clause+)? row_movement_clause? flashback_archive_clause?
```

Comment:
Could you please add line breaks for better readability to rules with such long strings? Decide appropriate locations by yourself. Recommended line length is 80, max is 120. Your rule length is 440, so it's too long.